src/Sentry.NLog/Sentry.NLog.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

Comment:
I was hoping we would get rid of ns2.1 if we can, but avoid adding it to more libraries. 
It's fair that the lib supports it and so does the Sentry core SDK. But the package is fairly lower in terms of usage: https://nugettrends.com/packages?months=72&ids=Sentry.NLog&ids=Sentry

And most people are updating their runtimes. .NET Standard 2.1 came to help bridge .NET Framework and .NET Core 3.1. We had .NET 5,6,7,8 now so hoping we can get rid of those netstandard

How about we remove this?

Reply:
NLog doesn't have ns2.1: https://www.nuget.org/packages/NLog#dependencies-body-tab